packages/wrangler/src/queues/cli/commands/consumer/add.ts

```diff
@@ -55,6 +64,8 @@ export async function handler(
 			max_wait_time_ms: args.batchTimeout // API expects milliseconds
 				? 1000 * args.batchTimeout
 				: undefined,
+			concurrency_enabled: args.concurrencyEnabled ?? true,
```

Comment:
Was concurrency previously defaulted to false/not supported?

Reply:
It was previously not supported at all, and we want it to be enabled by default going forward (with the ability to opt out).

Reply:
Concurrency is a new feature. We will default to true when this is not explicitly set to `false`.

Reply:
@elithrar I'm actually second guessing this now, sorry for the back and forth.

I was trying to be a bit forward looking by defaulting to `true` here in the CLI command. But we probably just want the CLI behavior to match the API behavior (default to `false`) and then both will be "cleaned up" at the same time when we decide to flip the default behavior for a missing value.

Reply:
How about:

- We give users time - 1 week or so as we're in beta - to set `max_concurrency` if they need to.
- After the deadline, we start respecting concurrency settings and actually scaling up. For most users, `max_concurrency = null` is implied and thus the maximum.

No need for `concurrency_enabled` at all? @jbw1991

Reply:
Yeah, this is a good tradeoff. We can get rid of `concurrency_enabled`.
For the next few weeks, the API will assume that a missing/null `max_concurrency` should fallback to a default value of 1.
After communicating the change and allowing some time, we'll change the default to 'unlimited'.
There will be no way to use 'unlimited' mode until the API change goes live. But this is a fair price to pay for the cleaner API. Users can just select the maximum possible limit in the meantime.

Reply:
w.r..t wrangler flags:

* `--max-concurrency=N` to set a cap
* `--no-max-concurrency` to "null out" the setting (passes an explicit null)

Reply:
We discussed offline, but the `--no-max-concurrency` is more relevant for a `queues consumer update` command, which doesn't exist yet.

Everything else should be good to go. `--max-concurrency=N` is supported